packages/framework/src/Form/Admin/Order/OrderFormType.php

```diff
@@ -190,151 +342,171 @@ public function buildForm(FormBuilderInterface $builder, array $options)
+                            new Constraints\NotBlank([
+                                'message' => 'Please enter city',
+                                'groups' => [self::VALIDATION_GROUP_DELIVERY_ADDRESS_SAME_AS_BILLING_ADDRESS],
+                            ]),
+                            new Constraints\Length(['max' => 100,
+                                'maxMessage' => 'City name cannot be longer than {{ limit }} characters',
+                                'groups' => [self::VALIDATION_GROUP_DELIVERY_ADDRESS_SAME_AS_BILLING_ADDRESS],
+                            ]),
+                        ],
+                    ])
+                    ->add('deliveryPostcode', TextType::class, [
+                        'label' => t('Postcode'),
+                        'required' => true,
+                        'constraints' => [
+                            new Constraints\NotBlank([
+                                'message' => 'Please enter zip code',
+                                'groups' => [self::VALIDATION_GROUP_DELIVERY_ADDRESS_SAME_AS_BILLING_ADDRESS],
+                            ]),
+                            new Constraints\Length([
+                                'max' => 30,
+                                'maxMessage' => 'Zip code cannot be longer than {{ limit }} characters',
+                                'groups' => [self::VALIDATION_GROUP_DELIVERY_ADDRESS_SAME_AS_BILLING_ADDRESS],
+                            ]),
+                        ],
+                    ])
+                    ->add('deliveryCountry', ChoiceType::class, [
+                        'label' => t('Country'),
+                        'required' => true,
+                        'choices' => $countries,
+                        'choice_label' => 'name',
+                        'choice_value' => 'id',
+                        'constraints' => [
+                            new Constraints\NotBlank(['message' => 'Please choose country']),
+                        ],
+                    ])
+            );
+
+        return $builderShippingAddressGroup;
     }
 
     /**
-     * @param \Symfony\Component\OptionsResolver\OptionsResolver $resolver
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     * @return \Symfony\Component\Form\FormBuilderInterface
      */
-    public function configureOptions(OptionsResolver $resolver)
+    private function createNoteGroup(FormBuilderInterface $builder)
     {
-        $resolver
-            ->setRequired('order')
-            ->setAllowedTypes('order', Order::class)
-            ->setDefaults([
-                'data_class' => OrderData::class,
-                'attr' => ['novalidate' => 'novalidate'],
-                'validation_groups' => function (FormInterface $form) {
-                    $validationGroups = [ValidationGroup::VALIDATION_GROUP_DEFAULT];
+        $builderNoteGroup = $builder->create('noteGroup', GroupType::class, [
+            'label' => t('Contact us'),
```

Comment:
You've switched the labels by mistake:
![image](https://user-images.githubusercontent.com/10008612/48121541-824dfc00-e275-11e8-8deb-2ab663ff6083.png)

See master:
![image](https://user-images.githubusercontent.com/10008612/48121552-8aa63700-e275-11e8-86d8-bcd1d9bad775.png)

Reply:
done